src/xrpld/ledger/detail/View.cpp

```diff
@@ -1660,4 +1977,25 @@ deleteAMMTrustLine(
     return tesSUCCESS;
 }
 
+TER
+rippleCredit(
+    ApplyView& view,
+    AccountID const& uSenderID,
+    AccountID const& uReceiverID,
+    STAmount const& saAmount,
+    bool bCheckIssuer,
+    beast::Journal j)
+{
+    return std::visit(
+        [&]<ValidIssueType TIss>(TIss const& issue) {
+            if constexpr (std::is_same_v<TIss, Issue>)
+                return rippleCreditIOU(
+                    view, uSenderID, uReceiverID, saAmount, bCheckIssuer, j);
+            else
+                return rippleCreditMPT(
+                    view, uSenderID, uReceiverID, saAmount, j);
```

Comment:
Should this branch assert that `checkIssuer` is false? Who are the callers interested in `checkIssuer=true`? Why?